src/param_spec.rs

```diff
@@ -37,20 +39,26 @@ impl ParamSpec {
         unsafe { from_glib((*self.to_glib_none().0).flags) }
     }
 
-    pub fn get_blurb(&self) -> String {
-        unsafe { from_glib_none(gobject_sys::g_param_spec_get_blurb(self.to_glib_none().0)) }
+    pub fn get_blurb(&self) -> &str {
```

Comment:
These functions should use the `pub fn get_blurb<'a>(&self) -> &'a str` pattern to free the output lifetime since we know the strings come from GQuarks and are valid for `'static`.

Reply:
I was wrong, this only applies to `get_name`. Nick and blurb are stored in the pspec.

Reply:
We could have a `pub fn get_static_blurb<'a>(&self) -> Option<&'a str>` which checks the `flags` for `G_PARAM_STATIC_BLURB` but I'm not sure that's worth the effort.

Reply:
That doesn't seem very useful, but yes let's do that for name.